package.json

```diff
@@ -6,12 +6,14 @@
     "babel-preset-env": "^1.5.2",
     "chalk": "^1.1.3",
     "glob": "^7.1.2",
+    "jest": "^20.0.4",
     "lerna": "^2.0.0-rc.5",
     "mkdirp": "^0.5.1",
     "rimraf": "^2.6.1"
   },
   "scripts": {
     "build-clean": "rimraf ./packages/*/build ./packages/*/build-browser ./packages/*/build-module",
-    "build": "node ./scripts/build.js"
+    "build": "node ./scripts/build.js",
+    "test": "jest"
```

Comment:
> I doubt pretest would run the ESLint task if Jest is in watch mode...

You can modify `scripts` to run lint when initializing watch mode with this change:

```js
 "scripts": {
    "build-clean": "rimraf ./packages/*/build ./packages/*/build-browser ./packages/*/build-module",
    "build": "node ./scripts/build.js",
    "lint": "eslint . --ignore-path .gitignore",
    "pretest": "npm run lint",
    "test": "jest",
    "watch": "npm test -- --watch"
  }
```

I'm not sure it is possible to re-trigger eslint on every watch mode run, but it should be okey.